modules/custom/d_commerce_products_list/d_commerce_product_list.install

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * @file
+ * Install, update, and uninstall functions for d_commerce_product_list.
+ */
+
+/**
+ * Implements hook_install().
+ */
+function d_commerce_product_list_install() {
+  $theme_name = \Drupal::config('system.theme')->get('default');
+  $config_service = \Drupal::service('config.factory');
+  $configs = d_commerce_product_list_get_block_configs();
+
+  foreach ($configs as $config) {
+    $config_service->getEditable($config)
+      ->set('theme', $theme_name)
+      ->set('dependencies.theme', $theme_name)
+      ->save();
+  }
+}
+
+/**
+ * Get all config names for blocks.
+ *
+ * @return array
+ *   List of configs names.
+ */
+function d_commerce_product_list_get_block_configs() {
+  $configs = [];
+  $dir = drupal_get_path('module', 'd_commerce_product_list');
+  $files = scandir($dir . '/config/optional');
+  foreach ($files as $file) {
+    if (strpos($file, 'block.block') !== FALSE) {
+      $configs[] = substr($file, 0, -4);
+    }
+  }
+  return $configs;
```

Comment:
aa widzisz, @tomasz-makiel-droptica , Grzegorz zwrócił na boku uwagę, że w obu tych install powtarza się ten kod od wyszukiwania configów i aktualizacji :point_up: :point_up:  można by to przenieść do jakiegoś service .. może w d_p ? i tu ograniczyć się tylko do install w którym się wywoła metodę? co myślisz ?

Reply:
dałbym raczej gdzieś do `d_content_init`, bo pewnie trzeba będzie kiedyś zrobić tę samą magię w `d_product`